R/utilities-enums.R

```diff
@@ -319,69 +319,71 @@ Linetypes <- enum(c(
-#' x = (seq_along(Shapes)-1) %% 6,
-#' y = floor((seq_along(Shapes)-1)/6),
-#' shape = factor(names(Shapes), levels = names(Shapes))
+#'   x = (seq_along(Shapes) - 1) %% 6,
+#'   y = floor((seq_along(Shapes) - 1) / 6),
+#'   shape = factor(names(Shapes), levels = names(Shapes))
 #' )
-#' ggplot2::ggplot(data = shapesData, ggplot2::aes(x, y)) + 
-#' ggplot2::theme_void() +
-#' # Define size and color of shapes
-#' ggplot2::geom_point(ggplot2::aes(shape = shape), size = 8, color = "red") +
-#' # Add shape names from enum below the displayed shape
-#' ggplot2::geom_text(ggplot2::aes(label = shape), nudge_y = -0.3, size = 3) + 
-#' # Use scale to display the actual shape
-#' ggplot2::scale_shape_manual(values = as.character(unlist(Shapes))) +
-#' # Remove the legend as the shape name is labelled below the shape
-#' ggplot2::guides(shape="none")
-#' 
+#' ggplot2::ggplot(data = shapesData, ggplot2::aes(x, y)) +
+#'   ggplot2::theme_void() +
+#'   # Define size and color of shapes
+#'   ggplot2::geom_point(ggplot2::aes(shape = shape), size = 8, color = "red") +
+#'   # Add shape names from enum below the displayed shape
+#'   ggplot2::geom_text(ggplot2::aes(label = shape), nudge_y = -0.3, size = 3) +
+#'   # Use scale to display the actual shape
+#'   ggplot2::scale_shape_manual(values = as.character(unlist(Shapes))) +
+#'   # Remove the legend as the shape name is labelled below the shape
+#'   ggplot2::guides(shape = "none")
+#'
 #' # Perform a scatter plot with blue pentagons as shape
 #' addScatter(
-#' x = 1:10,
-#' y = rlnorm(10),
-#' shape = Shapes$pentagon,
-#' color = "blue",
-#' size = 3
+#'   x = 1:10,
+#'   y = rlnorm(10),
+#'   shape = Shapes$pentagon,
+#'   color = "blue",
+#'   size = 3
 #' )
-#' 
+#'
 Shapes <- list(
   # Usual symbols
+  ## Plain symbols
   "circle" = "\u2b24",
-  "circleOpen" = "\ud83d\udf85",
   "diamond" = "\u2bc1",
-  "diamondOpen" = "\ud83d\udf54",
   "triangle" = "\u25b2",
-  "triangleOpen" = "\ud83d\udf02",
-  "square" =  "\u25a0",
-  "squareOpen" = "\u25a1",
+  "square" = "\u25a0",
   "invertedTriangle" = "\u25bc",
-  "invertedTriangleOpen" = "\ud83d\udf04",
   "cross" = "\ud83d\udfad",
-  "thinCross" = "\ud83d\udfa9",
+  "star" = "\u2605",
+  "pentagon" = "\u2b1f",
+  "hexagon" = "\u2b22",
   "plus" = "\ud83d\udfa6",
-  "thinPlus" = "\ud83d\udfa2",
   "asterisk" = "\ud83d\udfbc",
-  # Weird ggplot2 bug, 
-  # the symbol does not show up for a handful of size
-  "star" = "\ud83d\udfca",
-  "starOpen" = "\u2606",  
-  "pentagon" = "\u2b1f",
+  ## Open/alt symbols
+  "circleOpen" = "\ud83d\udf85",
+  "diamondOpen" = "\ud83d\udf54",
+  "triangleOpen" = "\ud83d\udf02",
+  "squareOpen" = "\u25a1",
+  "invertedTriangleOpen" = "\ud83d\udf04",
+  "starOpen" = "\u2606",
   "pentagonOpen" = "\u2b20",
-  "hexagon" = "\u2b22",
   "hexagonOpen" = "\u2b21",
+  "thinCross" = "\ud83d\udfa9",
+  "thinPlus" = "\ud83d\udfa2",
+  # Weird ggplot2 bug,
+  # the symbol does not show up for a handful of size
```

Comment:
is this comment still valid or it can be removed now?

Reply:
Still valid here.
FYI, I am currently testing leveraging a downloaded font in the package for the shapes and it seems to work really well.
The new font will fix that issue and even allow us to have consistent shape sizes